src/main/java/com/github/platymemo/alaskanativecraft/mixin/client/ClientPlayNetworkHandlerMixin.java

```diff
@@ -25,17 +25,19 @@
             cancellable = true,
             locals = LocalCapture.CAPTURE_FAILHARD
     )
-    private void onEntitySpawn(EntitySpawnS2CPacket packet, CallbackInfo ci, double x, double y, double z, EntityType<?> type) {
+    private void onEntitySpawn(EntitySpawnS2CPacket packet, CallbackInfo ci, EntityType<?> type) {
```

Comment:
Actually, due to 1.17 changes, this whole mixin can be removed (thank god)

Reply:
Verified and removed in 207644bd7bd01e5a41841c25c95562144b62c9a1